### DISCUSSION TITLE
intellisense / autocomplete / type information for python variables in (jinja2) templates

### DISCUSSION THREAD

**Discussion by jgehrcke** [microsoft/pylance-release#4090] [Enhancement]
Hello! Given the beauty and richness of this extension I was a little surprised that within a jinja2 template there is not a lot of support in terms of type information and auto-completion (for the Python names in the template). I understand that this might be challenging to build (in particular because from within the template it might be completely undefined _how_ the variables referred to in the template are being populated), but I would love to hear the expert's opinion on this. In my special case there is a one-to-one mapping between template files and call sites, so in theory the information is unambiguous.

**Comment by rchiodo** [comment]
Thanks for the suggestion. We're actually discussing how we might implement this at the moment. It's good to know it's actually wanted.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
